lib/event_sourcery/postgres/event_store.rb

```diff
@@ -86,7 +86,7 @@ def latest_event_id(event_types: nil)
       # @return [Array] of found events
       def get_events_for_aggregate_id(aggregate_id)
         events_table.where(aggregate_id: aggregate_id.to_str).order(:version).map do |event_hash|
-          build_event(**event_hash)
+          build_event(event_hash)
```

Comment:
FTR I removed the `**` from here because I think it makes this PR a little less confusing, it still built fine before I removed these though 🤷